comp_sci_gender_bias/utils/process_pandas.py

```diff
@@ -23,3 +23,59 @@ def remove_nonalphanum_lowercase(column: pd.Series) -> pd.Series:
         .str.replace("[^a-z0-9 ]", "", regex=True)
         .str.strip()
     )
+
+
+def remove_fw_slash(df: pd.DataFrame, col: str) -> pd.DataFrame:
+    """Remove '/' from the end of all rows in specified
+    df and col"""
+    df[col] = df[col].map(lambda x: str(x)[:-1] if str(x)[-1] == "/" else x)
```

Comment:
My first suggestion was going to be:
```suggestion
    df[col] = df[col].map(lambda x: str(x)[:-1] if str(x).endswith("/") else x)
```
but actually I think will work and is neater
```suggestion
    df[col] = df[col].str.rstrip("/")
```

Reply:
Much simpler 🙂